### PR TITLE
Fix hexo 3.3.1

### PR DESCRIPTION
**Pull request by wafer-li**
Fix hexo 3.x not working.

Test on hexo 3.3.1

image
image

### CODE REVIEW

**Comment by wafer-li** [comment]
By the way, #4 is not working on 3.3.1

File: lib/generator.js [hunk 1]
```diff
@@ -32,22 +34,32 @@
...
-    generateTagsJson(locals.tags.toArray());
-    return callback();
+    return {
+        path: 'tags.json',
+        data: JSON.stringify(tagsMeta)
+    };
   };
 
-}).call(this);
+  var jsons = new Array();
```
Comment (wafer-li) [inline lib/generator.js]: This line may not be elegant.

Since I don't have much knowledge of JS, if there is any elegant way, please inform me.

**Comment by borgnix** [comment]
This seems cool! :)

**Comment by youngjuning** [comment]
@wafer-li Thanks for your fixed.

**Comment by wangray** [comment]
@timnew can you merge this? Thanks!

**Pull request merged**